doc/content/getting-started/upgrading/_index.md

```diff
@@ -120,6 +120,10 @@ Restore from the backup.
 root@:26257/defaultdb>  RESTORE DATABASE ttn_lorawan FROM 'nodelocal://self/<backup_name>';
 ```
 
+#### Postgres
+
+Please refer to the [official postgres documentation](https://www.postgresql.org/docs/14/backup.html) for the specific version that you are using.
```

Comment:
I prefer using "PostgreSQL" in text over "postgres", but feel free to keep as is if you prefer this way